README.md

```diff
@@ -252,15 +252,15 @@ Allows creation of a new IIS Application Pool and configuration of application p
 
 #### `ensure`
 
-Must be either 'present' or 'absent'. Present will ensure the application pool is created.
+Specifies whether an application pool should be present or absent. If `state` is not specified, the application pool will be created and left in the default started state.
 
 #### `name`
 
-Name of the application pool.
+The unique name of the ApplicationPool.
 
 #### `state`
 
-The state of the application pool. Must be either 'started' or 'stopped'.
+The state of the ApplicationPool. By default, a newly created application pool will be started
```

Comment:
Suggest re-including that the value must be either started or stopped in addition to the new information.

Reply:
Agree to the fullest extent of the law.